packages/cli-kit/src/output.ts

```diff
@@ -307,60 +311,94 @@ export const clearCollectedLogs = () => {
+ * @param logFunc {Function} The logging function to use to output to the user.
  */
-export const info = (content: Message) => {
+export const info = (content: Message, logFunc: Logger = consoleLog) => {
+  const message = stringifyMessage(content)
   if (isUnitTest()) collectLog('info', content)
-  message(content, 'info')
+  outputWhereAppropriate('info', logFunc, message)
 }
 
 /**
  * Outputs a success message to the user.
- * Success message receive a special formatting to make them stand out in the console.
+ * Success messages receive a special formatting to make them stand out in the console.
  * Note: Success messages are sent through the standard output.
  * @param content {string} The content to be output to the user.
+ * @param logFunc {Function} The logging function to use to output to the user.
  */
-export const success = (content: Message) => {
+export const success = (content: Message, logFunc: Logger = consoleLog) => {
   const message = colors.bold(`✅ Success! ${stringifyMessage(content)}.`)
   if (isUnitTest()) collectLog('success', content)
-  outputWhereAppropriate('info', consoleLog, message)
+  outputWhereAppropriate('info', logFunc, message)
+}
+
+/**
+ * Outputs a started message to the user.
+ * Started messages receive a special formatting to make them stand out in the console.
+ * Note: Started messages are sent through the standard output.
+ * @param content {string} The content to be output to the user.
+ * @param logFunc {Function} The logging function to use to output to the user.
+ */
+export const started = (content: Message, logFunc: Logger = consoleLog) => {
+  const message = `${colors.yellow('❯')} ${stringifyMessage(content)}`
+  if (isUnitTest()) collectLog('started', content)
+  outputWhereAppropriate('info', logFunc, message)
 }
 
 /**
  * Outputs a completed message to the user.
  * Completed message receive a special formatting to make them stand out in the console.
  * Note: Completed messages are sent through the standard output.
  * @param content {string} The content to be output to the user.
+ * @param logFunc {Function} The logging function to use to output to the user.
  */
-export const completed = (content: Message) => {
+export const completed = (content: Message, logFunc: Logger = consoleLog) => {
   const message = `${colors.green('✔')} ${stringifyMessage(content)}`
   if (isUnitTest()) collectLog('completed', content)
-  outputWhereAppropriate('info', consoleLog, message)
+  outputWhereAppropriate('info', logFunc, message)
+}
+
+/**
+ * Outputs a failed message to the user.
+ * Failed messages receive a special formatting to make them stand out in the console.
+ * Note: Failed messages are sent through the standard output.
+ * @param content {string} The content to be output to the user.
+ * @param logFunc {Function} The logging function to use to output to the user.
+ */
+export const failed = (content: Message, logFunc: Logger = consoleLog) => {
```

Comment:
Nitpick: I'd rename `logFunc` to `logger` in all the functions to align with the interface name `Logger`.